yaml_edit/lib/src/editor.dart

```diff
@@ -360,8 +360,8 @@ class YamlEditor {
   /// doc.spliceList([], 1, 0, ['Feb']); // [Jan, Feb, March, April, June]
   /// doc.spliceList([], 4, 1, ['May']); // [Jan, Feb, March, April, May]
   /// ```
-  Iterable<YamlNode> spliceList(Iterable<Object> path, int index,
-      int deleteCount, Iterable<Object> values) {
+  Iterable<YamlNode> spliceList(Iterable<Object?> path, int index,
+      int deleteCount, Iterable<Object?> values) {
     ArgumentError.checkNotNull(path, 'path');
     ArgumentError.checkNotNull(index, 'index');
     ArgumentError.checkNotNull(deleteCount, 'deleteCount');
```

Comment:
I would like to remove these `ArgumentError.checkNotNull()` calls as well since they're already handled by Dart.

Reply:
Yes, this is one of the nice things we can now do :D